contrib/debian/fwupd-qubes-vm-whonix.install

```diff
@@ -0,0 +1,2 @@
+usr/share/qubes-fwupd/fwupd_download_updates.py
+usr/share/qubes-fwupd/fwupd_common_vm.py
```

Comment:
consider moving these to `/usr/bin` or `/bin` instead

Reply:
I moved those scripts to the `/usr/libexec`. The VM scripts are internals that should be run only by `qubes-fwupdmgr`.